main.c

```diff
@@ -212,6 +213,20 @@ int main(int argc, char *argv[])
         INFO("kibosh_main: configured %s.\n", conf_str);
     }
 
+    /* Reset random seed for the process. */
+    if (conf->random_seed) {
+        srand(conf->random_seed);
+        INFO("kibosh_main: random seed is set to %d.\n", conf->random_seed);
+    } else {
+        int s = (int) round(time(0));
+        srand(s);
+        INFO("kibosh_main: random seed is set to %d.\n", s);
+    }
+
+    /* Start a process to clear page cache every 5 seconds. */
+    int sret = system("while true; do sleep 5; sudo sh -c \"echo 1 > /proc/sys/vm/drop_caches\"; done &");
```

Comment:
Could you please explain why do we need to do this ? A comment will be good.

Reply:
In order to make Kibosh compatible with Kafka, we had to remove the direct_io option, which allows Kibosh to retain cache.
This leads to two consequences:
1. page cache might build up
2. for read_delay fault, only the first read call is affected by the fault, the subsequent read calls can access the cache instead.
So the process is intended to clear the page cache periodically to mitigate those two problems.

Reply:
@yyu1993 could you please add this comment in the code?

Reply:
Sure